mapswipe_workers/mapswipe_workers/project_types/arbitrary_geometry/grouping_functions.py

```diff
@@ -65,50 +61,22 @@ def group_input_geometries(input_geometries_file, group_size):
         try:
             groups[group_id_string]
         except KeyError:
-            # todo: change this and call feature_geoms -> geojson
-            groups[group_id_string] = {
-                "feature_ids": [],
-                "feature_geometries": [],
-                "center_points": [],
-                "reference": [],
-                "screen": [],
-            }
+            groups[group_id_string] = {"feature_ids": [], "feature": []}
 
         # we use a new id here based on the count
         # since we are not sure that GetFID returns unique values
         groups[group_id_string]["feature_ids"].append(feature_count)
-        groups[group_id_string]["feature_geometries"].append(
-            json.loads(feature.GetGeometryRef().ExportToJson())
-        )
-
-        # from gdal documentation GetFieldAsDouble():
-        # OFTInteger fields will be cast to double.
-        # Other field types, or errors will result in a return value of zero.
-        center_x = feature.GetFieldAsDouble("center_x")
-        center_y = feature.GetFieldAsDouble("center_y")
-
-        # check if center attribute has been provided in geojson
-        # normal tasks will never have a center of 0.0, 0.0
-        # this is just in the middle of the ocean
-        if (center_x == 0.0) and (center_y == 0.0):
-            groups[group_id_string]["center_points"].append(None)
-        else:
-            groups[group_id_string]["center_points"].append([center_x, center_y])
 
         # this is relevant for the tutorial
-        reference = feature.GetFieldAsDouble("reference")
-        screen = feature.GetFieldAsDouble("screen")
-        groups[group_id_string]["reference"].append(reference)
-        groups[group_id_string]["screen"].append(screen)
+        if "screen" in feature.keys():
```

Comment:
I'm not sure if this is going to work. The feature here is the geojson feature, right? Did you test this explicitly with a file that contains these columns in the geojson file?

Reply:
true, the whole part is unnecessary, no error because it is not executed and screen and reference are already in properties by default